.github/workflows/ci-security.yml

```
@@ -0,0 +1,208 @@
+      - name: Restore pnpm store cache
+        id: pnpm-cache
+        timeout-minutes: 5
+        uses: actions/cache@v4
+        with:
+          path: ${{ steps.pnpm-setup.outputs.store_path }}
+          key: pnpm-store-${{ runner.os }}-node-${{ steps.node-version.outputs.range }}-${{ hashFiles('pnpm-lock.yaml') }}
+          restore-keys: |
+            pnpm-store-${{ runner.os }}-node-${{ steps.node-version.outputs.range }}-
+
+      - name: Register remote cache key
+        run: |
```

actionlint (1.7.8)

undefined function "replace". available functions are "always", "cancelled", "contains", "endswith", "failure", "format", "fromjson", "hashfiles", "join", "startswith", "success", "tojson"

(expression)